spec/support/loss_report_samples_helper.rb

```diff
@@ -0,0 +1,840 @@
+module Helpers
+  module LossReportSamplesHelper
+    def dif_id
+      'C1200000031-SEDAC'
+    end
+
+    def iso_id
```

Comment:
1. As mentioned previously, I think since this is static information, it should be moved to the fixtures folder
2. I think that hard coding in a concept id makes the tests brittle. While it is unlikely that they will change unless our cmr start up script changes, it is still possible. And if that script changes, someone would need to know to make the change here. Sometimes that can't be avoided, but if we can we should try and either try and use a method to grab the concept id or if we are in feature tests, we should do a CMR search and use that information (I am adding a comment in the feature test)
3. if we still need to use hard coded concept ids, I think having a tad more information in the method name would be helpful (e.g., loss_report_iso_id or something with more information)